src/Http/Productions/ProductionsSearchController.php

```diff
@@ -24,17 +26,39 @@ public function __construct(ProductionRepository $repository)
 
     public function search(Request $request): Response
     {
-        $keyword = $request->get('name');
+        $keyword = $request->get('name', '');
+        $start = (int) $request->get('start', self::DEFAULT_START);
+        $limit = (int) $request->get('limit', self::DEFAULT_LIMIT);
+        $pageNumber = (int) ($start / $limit);
```

Comment:
Won't this result in the first page number being `0`?

Reply:
Yes. I was a bit confused about it as well but I copied it from https://github.com/cultuurnet/udb3-silex/blob/108511132837398c79b488fb636a9b0adba13f4f/src/Http/Label/ReadRestController.php#L110 and assumed that we just started at 0.

However, I checked and now see that in SAPI3 we do the same but increase the value by `1` so the first page number is 1: https://github.com/cultuurnet/udb3-search-service/blob/master/src/Http/ResultTransformingPagedCollectionFactory.php#L44

So we're not consistent in whether it should be 0 or 1.

Additionally,  this info isn't even actually included in the response. For example, a collection of offers from SAPI3 on acc: https://udb-silex-acc.uitdatabank.be/offers/?disableDefaultFilters=true&apiKey=deb306a6-6f46-4c98-89ce-b03ec4fd11e2

```
{
  "@context": "http://www.w3.org/ns/hydra/context.jsonld",
  "@type": "PagedCollection",
  "itemsPerPage": 30,
  "totalItems": 428540,
  "member": [...]
}
```

It's only used if we would also inject a URL generator service, so it could generate a URL to the next and previous pages: https://github.com/cultuurnet/Hydra/blob/master/src/PagedCollection.php#L178

If you want I can change it to start from 1, but maybe it would be even better to not use this `PagedCollection` object at all and just hardcode this response object's properties to start removing a dependency on https://github.com/cultuurnet/hydra?

I don't think we're actually generating those previous and next URLs in most places (e.g. here for productions, labels search, sapi3, ...)

Reply:
In that case I would indeed suggest to use a custom response instead and either:
- let the UI worry about generating prev and next urls
- provider `prev` and `next` URLs from the backend in the response

Reply:
I added an internal `PagedCollectionResponse` with a simplified signature.

I've also refactored the two only other usages of `CultuurNet\Hydra\PagedCollection` (one in a controller, the other in the test for that controller) and removed the `cultuurnet/hydra` package from the composer dependencies.